.env.example

```
@@ -1,2 +1,9 @@
 RELATION_DATABASE_USERNAME=root
-RELATION_DATABASE_PASSWORD=qwer1234!@
\ No newline at end of file
+RELATION_DATABASE_PASSWORD=qwer1234!@
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RELATION_DATABASE_PASSWORD key should go before the RELATION_DATABASE_USERNAME key

(UnorderedKey)